client/aws/clients.go

```diff
@@ -0,0 +1,104 @@
+package aws
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/credentials/stscreds"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/autoscaling"
+	"github.com/aws/aws-sdk-go/service/cloudformation"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/aws/aws-sdk-go/service/ec2/ec2iface"
+	"github.com/aws/aws-sdk-go/service/elb"
+	"github.com/aws/aws-sdk-go/service/elb/elbiface"
+	"github.com/aws/aws-sdk-go/service/iam"
+	"github.com/aws/aws-sdk-go/service/iam/iamiface"
+	"github.com/aws/aws-sdk-go/service/kms"
+	"github.com/aws/aws-sdk-go/service/kms/kmsiface"
+	"github.com/aws/aws-sdk-go/service/route53"
+	"github.com/aws/aws-sdk-go/service/s3"
+	"github.com/aws/aws-sdk-go/service/s3/s3iface"
+	"github.com/aws/aws-sdk-go/service/sts"
+	"github.com/aws/aws-sdk-go/service/sts/stsiface"
+	"github.com/aws/aws-sdk-go/service/support"
+	"github.com/aws/aws-sdk-go/service/support/supportiface"
+	"github.com/giantswarm/microerror"
+)
+
+const (
+	// trustedAdvisorRegion describes the AWS region in which the trusted advisor
+	// service is available.
+	trustedAdvisorRegion = "us-east-1"
```

Comment:
I would be good to have this configurable via a flag IMO

Reply:
It would be good to have a use case for this. We can certainly implement such a feature when we need it, but not now in a boyscouting PR.

Reply:
It would be good to have a use case for this. We can certainly implement such a feature when we need it, but not now in a boyscouting PR.

Reply:
> It would be good to have a use case for this.

I would be even against making it a flag as long as we don't use this flag. That gives us a freedom to remove/change it any time.

Reply:
> It would be good to have a use case for this.

I would be even against making it a flag as long as we don't use this flag. That gives us a freedom to remove/change it any time.

Reply:
Trusted advisor only runs in this region